### PR TITLE
Onboard Kueue into CloMonitor

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
`#SecuritySlam`

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
@tegioz

**Comment by ArangoGutierrez** [comment]
cc @alculquicondor

File: data/cncf.yaml [hunk 1]
```diff
@@ -3426,3 +3428,17 @@
       check_sets:
         - community
         - code
+- name: kueue
+  display_name: kueue
+  description: Kubernetes-native Job Queueing.
+  category: runtime
+  logo_url: https://raw.githubusercontent.com/kubernetes-sigs/kueue/main/site/static/images/logo.svg
+  devstats_url: https://kueue.devstats.cncf.io/
```
Comment (alculquicondor) [inline data/cncf.yaml:3436]: Is this page going to be live once this PR merges?

Comment (ArangoGutierrez) [reply data/cncf.yaml:3436]: see example https://kcl.devstats.cncf.io/

File: data/cncf.yaml [hunk 2]
```diff
@@ -3426,3 +3428,17 @@
...
+  accepted_at: "2022-04-12"
+  maturity: graduated
```
Comment (kerthcet) [inline data/cncf.yaml:3438]: So for sig subprojects, it's always **graduated**, right? Unlike CNCF projects.

**Comment by tegioz** [comment]
Thanks @ArangoGutierrez! Kueue has been added to CLOMonitor as part of #1385.

**Pull request closed without merging**